src/style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, serif;
+
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --button-buy-color: #00acdc;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  height: 408px;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  margin-bottom: 4px;
+}
+
+.card__name-text {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-text-color);
+}
+
+.card__code {
+  margin-bottom: 16px;
+}
+
+.card__code-text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-text-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  margin: 0 0 16px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-text-color);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-text-color);
+}
+
+.card__buy {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  height: 40px;
+  background: var(--button-buy-color);
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  align-self: center;
+  text-transform: uppercase;
+  margin-bottom: 16px;
+  border: 1px solid var(--button-buy-color);
+}
+
+.card__buy:hover {
+  background: #fff;
+  color: var(--button-buy-color);
+}
+
+.star {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 24px;
+}
+
+.star__stars {
+  display: flex;
+  justify-content: center;
+  align-self: flex-start;
+}
+
+.star__one-star {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) 0 50% no-repeat;
+}
+
+.star__one-star:not(:last-of-type) {
+  margin-right: 4px;
+}
+
+.star__one-star:nth-child(-n+4) {
+  background: url(images/star-active.svg) 0 50%  no-repeat;
+}
+
+.star__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 18px;
+  color: var(--main-text-color);
+  text-decoration: none;
+}
```

Comment:
You should reuse the `stars `block from the `stars `task

Reply:
and place each block in a separate file

Reply:
explain more please, Agniya Ivanishyna said to remove these blocks